lib/subject_extensions.rb

```diff
@@ -1,11 +1,21 @@
+module WithDefaultExpectations
+  def default_expectations
+    [{ 'binding' => 'program', 'inspection' => 'Not:HasBinding' }, { 'binding' => "#{@name}", 'inspection' => 'HasBinding' }]
```

Comment:
Please put one expectation per line; it is easier to read. 

You could also extract it to another file